pkg/operator/ceph/csi/cluster_config.go

```diff
@@ -115,6 +115,8 @@ func updateNetNamespaceFilePath(clusterNamespace string, cc csiClusterConfig) {
 
 	if IsHolderEnabled() {
 		for _, centry := range cc {
+			// update netNamespaceFilePath only for CephCluster resources whose ClusterID and
+			// Namespace are the same (not for radosnamespaces, cephfssubvolumegroups, or others)
```

Comment:
sorry i didn't get this comment, it needs to be updated for all the clusterID's where the namespace name is matching.

Reply:
This is just recording the behavior that Praveen explained to me. This part of the loop only applies to clusters whose ClusterID and Namespace are identical.